src/Client/ClientRegistry.php

```diff
@@ -10,17 +10,26 @@
 
 namespace KnpU\OAuth2ClientBundle\Client;
 
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
-class ClientRegistry
+class ClientRegistry implements ContainerAwareInterface
 {
-    private $container;
+    use ContainerAwareTrait;
```

Comment:
This also changes `$container` from private to protected, which I don't want... until someone has a use-case :)